### PR TITLE
chore(*): rename package to Kinase

### PR DESCRIPTION
**Pull request by jrbotros**

### CODE REVIEW

File: src/redux/reducers/main.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import {
   UPDATE_FIELD,
 } from 'redux/constants';
 
-const defaultContextKey = 'tentDefault';
+const defaultContextKey = 'kinaseDefault';
```
Comment (marcua) [inline src/redux/reducers/main.js]: don't put the project name in the key?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# Tent
-Tent is an extension for Google Chrome allowing users to map web content to a list of annotations.
+# Kinase
```
Comment (marcua) [inline README.md]: Add a pronunciation key :)

File: src/redux/reducers/main.js (hunk 1 above)
Comment (jrbotros) [inline src/redux/reducers/main.js]: I wanted some way to make it clear that this was an internal name but can just call it `default` if you think that's better

File: README.md (hunk 2 above)
Comment (jrbotros) [inline README.md]: ✅

File: src/redux/reducers/main.js (hunk 1 above)
Comment (jrbotros) [inline src/redux/reducers/main.js]: Changed to `__default` so it's more clear that it's an internal name

**Pull request merged**